pkg/source/openstack/client.go

```diff
@@ -361,6 +371,10 @@ func (c *Client) GenerateVirtualMachine(vm *migration.VirtualMachineImport) (*ku
 		},
 	}
 
+	if vmObj.Description != "" {
+		newVM.ObjectMeta.Annotations[annotationDescription] = vmObj.Description
```

Comment:
Do you think we need to ensure the `Annotations` field is not `nil` in the first place?

Reply:
You are right. `Annotations` are not initialized at this time.